packages/components/jest.setup.ts

```diff
@@ -1 +1,19 @@
 import "@testing-library/jest-dom";
+import { format } from "util";
+
+/**
+ * Ensure `console.error` calls throw an error in tests
+ */
+function throwOnConsoleError() {
+  console.error = function (...args: unknown[]) {
+    const [firstArg, ...restArgs] = args;
+
+    if (firstArg instanceof Error) {
+      throw firstArg;
+    }
+
+    throw new Error(format(firstArg, ...restArgs));
```

Comment:
Nice thing about this is it would've prevented the nested `<p>` tag console log we had for the longest time.